test/e2e/data/infrastructure-provider-byoh/v1beta1/templates/docker/clusterclass-quickstart.yaml

```
@@ -201,12 +194,12 @@ apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
 kind: KubeadmConfigTemplate
 metadata:
   name: quickstart-worker-bootstrap-template
-  namespace: ${NAMESPACE}
 spec:
   template:
     spec:
       joinConfiguration:
         nodeRegistration:
           kubeletExtraArgs:
             cgroup-driver: cgroupfs
-            eviction-hard: nodefs.available<0%,nodefs.inodesFree<0%,imagefs.available<0%
\ No newline at end of file
+            eviction-hard: nodefs.available<0%,nodefs.inodesFree<0%,imagefs.available<0%
+---
```

nit: a new line is required here if in some cases these files will be merged into a single YAML.
```suggestion
---

```